jax/experimental/jax2tf/tests/jax2tf_test.py

```diff
@@ -516,6 +516,15 @@ def jax_fn_array(x):
         tf_fn_array(np.array([3, 4, 5])), np.array([4.5, 10, 17.5],
                                                    jnp.bfloat16))
 
+  def test_kwargs(self):
+    # Re: https://github.com/google/jax/issues/6791
+    def f_jax(x):
```

Comment:
Would this test fail if we replace `x` with `y`? Probably not. Maybe the test would be clearer if `x` is forced to be a kwarg here?

Reply:
I've now forced `x` to be a `kwarg` by adding a `*`.